karmada-CLI-installtion-and-deploy-example/intro.md

```diff
@@ -0,0 +1,39 @@
+### background：
+
+1. The kubeconfig files for the Karmada cluster, member1 cluster, and member2 cluster are located in the $HOME/.kube directory. These files are named config, config-member1, and config-member2 respectively.
+
+   ```shell
+      $HOME/.kube/config
+      $HOME/.kube/config-member1
+      $HOME/.kube/config-member2
+   ```
+
+### install karmada through CLI tool
+
+1. install karmada CLI tool: karmadactl
+
+   ```shell
+   curl -s https://raw.githubusercontent.com/karmada-io/karmada/master/hack/install-cli.sh | sudo bash
+   ```
+2. initialize karamda
+
+   ```shell
+   karmadactl init
+   ```
+3. join member clusters（kind-member1 and kind-member2）to karmada cluster
+
+   ```shell
```

Comment:
I tried the example, but it requires manual input from you.
How about this one, just click,it will exec the command?
https://killercoda.com/kubevela/course/quick-start/kubevela-first-app
![image](https://user-images.githubusercontent.com/19745536/229458086-7fff464e-1fea-46d0-a318-a2cb8e151c2c.png)

Reply:
The MD file should specify that the subsequent operations can only be performed after the kind cluster is prepared.